packages/disploy/src/structs/GuildBan.ts

```diff
@@ -19,9 +19,12 @@ export class GuildBan extends Base {
 	 * The reason for this ban.
 	 */
 	public reason!: string | null;
-
+	/**
+	 * The raw data of the ban.
+	 */
+	 declare public raw: APIBan;
 	public constructor(app: App, { raw, guild }: { raw: APIBan; guild: Guild }) {
-		super(app);
+		super(app, raw);
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.